testing/src/scenario/mocking.py

```diff
@@ -2,6 +2,11 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+"""Juju and Pebble mocking
+
+This module contains mocks for the Juju and Pebble APIs that are used by ops
+to interact with the Juju controller and the Pebble service manager."""
+
```

Comment:
```suggestion
to interact with the Juju controller and the Pebble service manager.
"""

```

Yes, odd that the checker didn't find that.